parsing/padding.go

```diff
@@ -0,0 +1,113 @@
+package parsing
+
+import (
+	"errors"
+	"github.com/filecoin-project/go-data-segment/types"
+)
+
+// Pad pads a general byte array in to FP32 chunks of bytes where the topmost bits of the most significant byte are 0
```

Comment:
Nit: it isn't FP32 but Fr32, it gets its name from the `r` of BLS-12-381: https://github.com/filecoin-project/paired/tree/master/src/bls12_381#bls12-381-instantiation.
It is a padding scheme compatible with Field over `r`.

Reply:
Fixed in commit cea535e1000d13d5d4ffaf21eb601b3731553bee